src/consumer.cpp

```diff
@@ -73,9 +73,17 @@ Consumer::~Consumer() {
         rebalance_error_callback_ = nullptr;
         close();
     }
-    catch (const Exception&) {
-        // If close throws just silently ignore until there's some
-        // logging facility (if any)
+    catch (const Exception& ex) {
+        constexpr const char* library_name = "cppkafka";
+        std::ostringstream error_msg;
```

Comment:
Minor: use `using std::ostringstream` after `include` directives and remove the `std::` specifier here.